app/frontend/src/store/modules/form.js

```diff
@@ -3,6 +3,8 @@ import ipcService from '../../services/ipcService';
 export default {
   namespaced: true,
   state: {
+    getFormError: '',
+    gettingForm: false,
```

Comment:
Optional Style: Almost half tempted to say this should be "loadingForm" instead as it'd be slightly more clear what it's doing (loading) vs just getting.